skopt/learning/forest.py

```diff
@@ -3,6 +3,44 @@
 from sklearn.ensemble import ExtraTreesRegressor as sk_ExtraTreesRegressor
 
 
+def _return_std(X, trees, predictions):
+    """
+    Return standard deviation in predictions given a list of trees.
```

Comment:
By given a list of trees, I had meant `given a list of trees as a parameter in the programming sense` and not `std given a tree in the probabilistic sense` . My bad, that it implied that :P